modules/configuring-localnet-switched-topology.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-localnet-switched-topology_{context}"]
```

Comment:
putting this info here since I don't know where else to put it: I'm unsure if we should explain how to configure the overlay. I think so though - i.e. use the ovs bridge mappings to indicate the OVS bridges used for each network.

@trozet / @jcaamano thoughts ?...

Reply:
I guess that we did not properly plan for a user-friendly approach to this because we don't have a use case for localnet yet. 

So one option would be to not document it and not support it (without actually preventing its use).

Other option is to document it. You would have to log into each node and:

- Fetch the existing bridge-mappings: 
  `BRIDGE_MAPPINGS=$(ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-bridge-mappings)`

- Add the new bridge mapping:
  `BRIDGE_MAPPINGS="${BRIDGE_MAPPINGS},<network>:<bridge>"`
  
- Update the bridge mappings:
  `ovs-vsctl set Open_vSwitch . external_ids:ovn-bridge-mappings=${BRIDGE_MAPPINGS}`
  
I would add a disclaimer that it is a delicate operation, that all possible care should be taken to not override the existing bridge mappings that have already been added by the system and that it should be done in the shortest time and not while other sensible procedures are being applied to the system as to avoid a race condition where we would read some old bridge mappings overriding a possible modification performed by the system while this procedure is undertaken.

As @stevsmit says though it would be nice if this procedure was backed up by some upstream documentation. In the meantime, would this be enough information @stevsmit?

Reply:
@jcaamano I can volunteer to document this upstream.

... and I would rather keep this hidden, and not advertise it at all. 

> So one option would be to not document it and not support it (without actually preventing its use).

@jcaamano you mean omit the entire localnet "page" ?

Reply:
> @jcaamano you mean omit the entire localnet "page" ?

yes, that is what I meant.

> 
> ```shell
> BRIDGE_MAPPINGS="${BRIDGE_MAPPINGS},\<network_name\>_br-localnet:\<bridge\>"
> ```

wow, I actually had a comment on the localnet PR about why where we adding `_br-localnet` suffix. It was to prevent unexpected name collisions. I forgot to further pursue the comment in the heat of the moment to consider not using that suffix. 

Why are you escaping the `<`,`>` though? I meant that to be just a variable delimiter for the doc.

Reply:
+1 on omitting the entire page.

> Why are you escaping the <,> though? I meant that to be just a variable delimiter for the doc.

Sorry for the confusion ! You're right. I could swear the `Preview` tab of this didn't render it correctly. It should **not** be escaped in the bash script. (maybe I hit it when I didn't specify the snippet as bash ?... :thinking: )

For clarity, this is how it should look like:
```bash
BRIDGE_MAPPINGS="${BRIDGE_MAPPINGS},<network_name>_br-localnet:<bridge>"
```

Reply:
To be sure, are we suggesting I hide the entirety of [this section](https://56657--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-secondary-networks.html#configuring-localnet-switched-topology_configuring-secondary-networks) about configuring the localnet topology, correct? 

That's easy enough to simply mark out for now. If we have a use-case of it in the future, we could un-hide it and build upon it.